### ISSUE TITLE
Crash when disabling shaders again after enabling them

### ISSUE THREAD

**Issue by Colaskink** [IrisShaders/Iris#1731]
### Minecraft Version

1.19.2

### Iris Version

iris-mc1.19.2-1.4.0.jar

### Sodium Version

sodium-fabric-mc1.19.2-0.4.4+build.18.jar

### Operating System

Windows 10

### What is your GPU?

Nvidia Geforce RTX 2080

### Java Version

Java 16

### Reproduction Steps

1. Enable shaders (in this case complementary shaders)
2. Disable them
3. Crash

### Crash Report file and latest.log

```shell
https://gist.github.com/Colaskink/6422cde4ba5a2b6565145068b0b1b13d
https://gist.github.com/Colaskink/d9de7461c45ac098a6a12d54d3d4daea
```

### Additional context

Upon crashing, it does not generate any files in the "crash-reports" folder, but it does generate weird files in the main directory called "hs_err_pid3288.log" or something similar, which i have embedded too.

This is a modpack

**Comment by Lolothepro** [comment]
https://github.com/CaffeineMC/sodium-fabric/issues/1486 ?

**Comment by Colaskink** [comment]
> [CaffeineMC/sodium-fabric#1486](https://github.com/CaffeineMC/sodium-fabric/issues/1486) ?

Im currently using nvidia drivers 526.86, which may be the cause
I'll try downgrading them and see if it fixes it.
Also i removed the "hs_err_pid3288.log" gist due to sensitive information that i had not noticed earlier

**Comment by Colaskink** [comment]
Ended up removing litematica instead of downgrading drivers, and it fixed the issue. Thanks!

**Comment by coderbot16** [comment]
A workaround for this issue is now available in the latest versions of Iris 1.5.x and Sodium.
